pushing read me to new_updates

yashodalasith opened the pull request:
pushing read me to with detailed description and set up information

SayunHetti committed:
Initial commit on new branch

yashodalasith committed:
Initial commit on new branch

SayunHetti committed:
1st commit

yashodalasith committed:
testing branch

SayunHetti committed:
first message

yashodalasith committed:
Delete testinggitpush.txt

yashodalasith committed:
Backend and frontend foundation with css

yashodalasith committed:
Merge branch 'master' of github.com:yashodalasith/Textile-Management-System

SayunHetti committed:
Merge branch 'master' into sayun_payment

SayunHetti committed:
second

SayunHetti committed:
3rd

yashodalasith committed:
Merge branch 'master' into yashoda_discount

yashodalasith committed:
Connected to data base

yashodalasith committed:
Added admin dashboard for view and manage discount data

SayunHetti committed:
4th

yashodalasith committed:
created backend for applying discount and fetching data

SayunHetti committed:
payment front done but not conneceted

yashodalasith committed:
Added some css

SayunHetti committed:
6 th or 7th commit

yashodalasith committed:
Merge branch 'sayun_payment' into yashoda_discount

yashodalasith committed:
resolved conflicts

h-muhammed committed:
Create README.md

SayunHetti committed:
8 th commiit ig lol

IT22366368 committed:
backend CRUD

CRUD

IT22366368 committed:
changed backend

backend changed

IT22366368 committed:
BACKEND

BACKEND

yashodalasith committed:
Added some css

yashodalasith committed:
Merged and resolve conflicts

yashodalasith committed:
Fetched some data from backend

SayunHetti committed:
front end cart data fetch

SayunHetti committed:
Merge branch 'Samadhi' into sayun_payment

SayunHetti committed:
css front end cart and payment page

yashodalasith committed:
Added cors adn scheduled a task

yashodalasith committed:
changed backend to reset discount evry 24 hours

SayunHetti committed:
button issues fixed 70%

yashodalasith committed:
Resolved null value errors

yashodalasith committed:
Fix issues in frontend and least sales items

yashodalasith committed:
modify frontend and backend

SayunHetti committed:
Payment slip css

yashodalasith committed:
Added header and footer and some css

yashodalasith committed:
Added more css

SayunHetti committed:
loading effects

yashodalasith committed:
Added more css

IT22366368 committed:
new changes

yashodalasith committed:
fixed minor backend issues

yashodalasith committed:
merge samadhi brach and resolve conflicts

yashodalasith committed:
remove repeated footer

yashodalasith committed:
updated samadhi's backend

h-muhammed committed:
Yasho+sama+Mhd

h-muhammed committed:
Create README.md

SayunHetti committed:
Merge branch 'Samadhi' into sayun_payment

SayunHetti committed:
Home oooooooooooooooooooooooooooppp

h-muhammed committed:
report generate part added

yashodalasith committed:
space errors

yashodalasith committed:
added ai assistant

yashodalasith committed:
Formatted ai response

yashodalasith committed:
store discount time to remove them after an hour

SayunHetti committed:
loading effect in home

SayunHetti committed:
Merge branch 'MergedBranchMhd' of https://github.com/yashodalasith/Textile-Management-System into MergedBranchMhd

yashodalasith committed:
merge with sayun and resolve conflicts

yashodalasith committed:
minor changes

yashodalasith committed:
fixed some warnings

yashodalasith committed:
Fixed data fetching issues

yashodalasith committed:
fixed navigation issues

SayunHetti committed:
cart empty function message pop

SayunHetti committed:
merged with mohomed

SayunHetti committed:
minor error remaining

h-muhammed committed:
Update Login.jsx

h-muhammed committed:
Update index.css

yashodalasith committed:
report generation created

SayunHetti committed:
View all orders part added

SayunHetti committed:
resolved userId token Conflict

yashodalasith committed:
merged and resolved other branches

SayunHetti committed:
add styling to the report and the bills

yashodalasith committed:
added css to home page

yashodalasith committed:
some minor changes

yashodalasith committed:
added some css to home page

yashodalasith committed:
change discount percentage to 0 when removing discount

yashodalasith committed:
merged sayun branch changes

yashodalasith committed:
fixed a backend issue to apply discount correctly

IT22366368 committed:
make Style changes

yashodalasith committed:
minor changes

yashodalasith committed:
Merged samadhi branch and fixed css

yashodalasith committed:
improved chatbot and added more styling

yashodalasith committed:
merged mohommed branch

SayunHetti committed:
edit button on cart

SayunHetti committed:
Customer summary added

SayunHetti committed:
Name change Header

yashodalasith committed:
add about us page

yashodalasith committed:
merged sayun branch

yashodalasith committed:
added private routes

yashodalasith committed:
added error message when no itmes available

yashodalasith committed:
error messages added

yashodalasith committed:
merged yashoda branch

yashodalasith committed:
added button to navigate /products for admin

yashodalasith committed:
Merge branch 'yashoda_discount'

yashodalasith committed:
merged muhammed new updates branch to master

h-muhammed committed:
new changes

yashodalasith committed:
Merge branch 'new-updates'

yashodalasith committed:
add both pages to admin login

yashodalasith committed:
add private route alerts

yashodalasith committed:
added dashboard label

yashodalasith committed:
login navigation

yashodalasith committed:
Update README with Textile Management System details

Added detailed information about the Textile Management System, including features, prerequisites, installation instructions, and project structure.

IT22103840 commented on README.md:
````diff
@@ -1,2 +1,94 @@
+   npm install
+   ```
+
+3. *(If backend code is included, repeat the install steps in the backend directory as well.)*
````
please explain backend steps as well

yashodalasith committed:
Enhance README with sneak peek and installation steps to resolve comment

Added sneak peek section with demo images and improved installation instructions.

IT22103840 reviewed (approved):
approved
